docs/get-started/allocation/allocation-change-request.md

```diff
@@ -0,0 +1,53 @@
+# Request change to Resource Allocation to an existing project
+
+If past resource allocation is not sufficient for an existing project, PIs or project
+managers can request for change by clicking "Request Change" button on project
+resource allocation detail page as show below:
+
+![Request Change Resource Allocation](images/coldfront-request-change-allocation.png)
+
+This will bring up the detailed Quota attributes for that project as shown below:
+
+## Request Change Resource Allocation Attributes for OpenStack Project
+
+![Request Change Resource Allocation Attributes for OpenStack Project](images/coldfront-openstack-allocation-attributes.png)
+
+### Allocation Change Requests for OpenStack Project
```

Comment:
This title does not appear to match what is on lines 16-19, also I suggest finishing the resource allocation steps before saying that users can view the details, that should come after you finish explaining how to request allocations.

Reply:
I re-align the lines.